src/styles/card.css

```diff
@@ -0,0 +1,86 @@
+.card {
+  box-sizing: border-box;
```

Comment:
you can move box-sizing to a * selector, it may be easier for the future

Reply:
Thank you for comments. So should I make changes now or not?

Reply:
I made changes in the code and clicked re-request